app/views/views.py

```diff
@@ -204,7 +204,7 @@ def _context(title, override={}):
 
 
 def _error(request: HttpRequest, err_msg="Something went wrong.", e=None):
-    # logger.exception(err_msg)
+    # logger.exception(e)
```

Comment:
Feel free to delete the commented out logger

Reply:
I will leave it here ;) it was helpful.